y3prediction/prediction.py

```diff
@@ -771,17 +816,31 @@ def main(actx_class,
             print(f"\tWriting viz data exactly every {t_viz_interval} seconds.")
         print("#### Visualization setup: ####")
 
-    """
-    if not noslip:
-        vel_sigma = 0.
-    if adiabatic:
-        temp_sigma = 0.
-    """
+    if rank == 0:
+        print("\n#### Simluation initialization data: ####")
+        if init_case == "y3prediction":
+            print("\tInitializing flow to y3prediction")
+            print(f"\tInflow stagnation pressure {total_pres_inflow}")
+            print(f"\tInflow stagnation temperature {total_temp_inflow}")
+            print(f"\tInjection stagnation pressure {total_pres_inj}")
+            print(f"\tInjection stagnation temperature {total_temp_inj}")
+        elif init_case == "shock1d":
+            print("\tInitializing flow to shock1d")
+            print(f"Shock Mach number {mach}")
+            print(f"Ambient pressure {pres_bkrnd}")
+            print(f"Ambient temperature {temp_bkrnd}")
+        else:
+            from mirgecom.simutil import SimulationConfigurationError
+            raise SimulationConfigurationError(
+                "Invalid initialization configuration specified"
+                "Currently supported options are: "
+                "\t y3prediction"
+                "\t shock1d"
+            )
+        print("#### Simluation initialization data: ####")
 
     if rank == 0:
```

Comment:
Couldn't this just join the rank==0 block just above?  There are a few blocks that could join the rank == 0 block I think.  We should also comb through and make sure everything that can is coming through on rank 0, and rank > 0 only when appropriate and necessary.  Not a big deal for small nproc, but quite a stumble for 100's of ranks.